src/types/modules/irc-framework.d.ts

```diff
@@ -0,0 +1,429 @@
+/* eslint-disable no-use-before-define */
+// @ts-nocheck
+// eslint-disable
+
+// https://raw.githubusercontent.com/eternagame/HTML-Chat/vue-rewrite/src/app/types/modules/irc-framework/irc-framework.d.ts
+// TODO: Fix this
+declare module "irc-framework" {
```

Comment:
This file should eventually be moved to irc-framework itself.

Reply:
Yeah, I've been talking with itsonlybinary and will be porting irc-fw to typescript soon (tm)